[8.0.1.0.0][account_analytic_line_analysis]

azmimr67 opened the pull request:
Account Analytic Line Analysis
Reporting

azmimr67 committed:
[8.0.1.0.0][account_analytic_line_analysis]
Account Analytic Line Analysis
Reporting

andhit-r commented on account_analytic_line_analysis/reports/account_analytic_line_analysis.py:
```diff
@@ -0,0 +1,122 @@
+            a.id AS id,
+            a.unit_amount AS unit_amount,
+            a.amount AS amount,
+            a.name AS name,
+            a.ref AS ref,
```
Tidak perlu

andhit-r commented on account_analytic_line_analysis/reports/account_analytic_line_analysis.py:
```diff
@@ -0,0 +1,122 @@
+    amount = fields.Float(
+        string="Amount",
+    )
+    name = fields.Char(
+        string="Description",
+    )
+    ref = fields.Char(
+        string="Ref.",
```
Tidak perlu

andhit-r commented on account_analytic_line_analysis/reports/account_analytic_line_analysis.xml:
```diff
@@ -0,0 +1,88 @@
+            <field name="model">account.analytic_line_analysis</field>
+            <field name="arch" type="xml">
+                <search>
+                    <field name="name"/>
+                    <field name="ref"/>
```
Tidak perlu

andhit-r commented on account_analytic_line_analysis/reports/account_analytic_line_analysis.py:
```diff
@@ -0,0 +1,122 @@
+    	    a.company_id AS company_id,
+    	    a.product_id AS product_id,
+    	    a.general_account_id AS general_account_id,
+            b.id AS move_id
```
Tambahkan:
b.journal_id AS general_journal_id
b.period_id AS period_id

andhit-r commented on account_analytic_line_analysis/reports/account_analytic_line_analysis.xml:
```diff
@@ -0,0 +1,88 @@
+                    <field name="company_id"/>
+                    <field name="product_id"/>
+                    <field name="general_account_id"/>
+                    <field name="move_id"/>
```
Tambahkan pencarian berdasarkan:

1. general_journal_id
2. period_id

andhit-r commented on account_analytic_line_analysis/reports/account_analytic_line_analysis.xml:
```diff
@@ -0,0 +1,88 @@
+                        <filter name="row_grp_company" string="Company" context="{'group_by':'company_id'}" groups="base.group_multi_company"/>
+                        <filter name="row_grp_product" string="Product" context="{'group_by':'product_id'}"/>
+                        <filter name="row_grp_general_account" string="General Account" context="{'group_by':'general_account_id'}"/>
+                        <filter name="row_grp_move" string="Move Line" context="{'group_by':'move_id'}"/>
```
Tambahkan row grouping berdasarkan:

1. general_journal_id
2. period_id

andhit-r commented on account_analytic_line_analysis/reports/account_analytic_line_analysis.xml:
```diff
@@ -0,0 +1,88 @@
+                        <filter name="col_grp_company" string="Company" context="{'group_by':'company_id'}" groups="base.group_multi_company"/>
+                        <filter name="col_grp_product" string="Product" context="{'group_by':'product_id'}"/>
+                        <filter name="col_grp_general_account" string="General Account" context="{'group_by':'general_account_id'}"/>
+                        <filter name="col_grp_move" string="Move Line" context="{'group_by':'move_id'}"/>
```
Tambahkan column grouping berdasarkan:

1. general_journal_id
2. period_id

andhit-r reviewed (changes requested):
Jangan lupa menambahkan branding banner terbaru

azmimr67 committed:
[8.0.1.0.0][account_analytic_line_analysis]
Menghapus name dan ref
Menambah general_journal_id dan period serta menambah banner'
Menambah Banner

andhit-r reviewed (changes requested):
Access right belum ada

azmimr67 committed:
[8.0.1.0.0][account_analytic_line_analysis]
Revisi
Menambah hak access

andhit-r committed:
[account_analytic_line_analysis] Fix PEP8
